lib/Jit/jitoptions.cpp

```diff
@@ -161,6 +172,56 @@ ::DumpLevel JitOptions::queryDumpLevel(LLILCJitContext &Context) {
   return JitDumpLevel;
 }
 
+bool JitOptions::queryIsExcludeMethod(LLILCJitContext &JitContext) {
+  return queryMethodSet(JitContext, ExcludeMethodSet,
+                        (const char16_t *)UTF16("AltJitExclude"));
+}
+
+bool JitOptions::queryIsBreakMethod(LLILCJitContext &JitContext) {
```

Comment:
Can we make BreakMethod's name more specific as to where it stops so it's easier to disambiguate from the JitHalt semantics (inserted dbgbreak in the generated code)?